torch/csrc/jit/codegen/cuda/type.h

```diff
@@ -31,6 +31,8 @@ enum class ValType {
   NamedScalar,
 };
 
+enum class PredicateType { Value, Inline, Unswitch, Misaligned, InternalSync };
```

Comment:
What does the `Value` type mean?

Reply:
The `Predicate` class contains a `Bool` value. `PredicateType::Value` is used when you directly generate the `Bool` conditional without using a function from `PredicateCompute`. e.g.  the misaligned vectorization predicates